cpp/modmesh/python/common.hpp

```diff
@@ -505,21 +505,43 @@ class MODMESH_PYTHON_WRAPPER_VISIBILITY Interpreter
 // Suppress the warning "greater visibility than the type of its field"
 #pragma GCC diagnostic ignored "-Wattributes"
 #endif
-class PyStdErrOutStreamRedirect
+class PythonStreamRedirect
 {
 
 public:
-    PyStdErrOutStreamRedirect();
-    std::string stdout_string();
-    std::string stderr_string();
-    ~PyStdErrOutStreamRedirect() noexcept(false);
+
+    PythonStreamRedirect(bool enabled)
```

Comment:
Only constructor is needed, because of the introduction of `activate()` and `deactivate()`.